packages/block/src/block/constructors.ts

```diff
@@ -57,71 +59,25 @@ import type {
   WithdrawalBytes,
 } from '@ethereumjs/util'
 
-/**
- * Static constructor to create a block header from a header data dictionary
- *
- * @param headerData
- * @param opts
- */
-export function createBlockHeader(headerData: HeaderData = {}, opts: BlockOptions = {}) {
-  return new BlockHeader(headerData, opts)
-}
+export function normalizeTxParams(_txParams: any) {
```

Comment:
Two things on this function:
1) It isn't a constructor so ideally we put it in a helpers/utils file.  
2) We have essentially the same function [here](https://github.com/ethereumjs/ethereumjs-monorepo/blob/659d4280609943effe5322dcec7747a0cb6e50f8/packages/tx/src/fromRpc.ts#L5).  Can we harmonize these two and consolidate in one location (maybe in `@ethereumjs/util/helpers.ts` or `internal.ts`)  The only real difference I see is converting `0x0` to `0x` in the transaction signatuere values.